src/components/Database/DataViewer/NodeActions.tsx

```diff
@@ -131,24 +131,26 @@ export const NodeActions = React.memo<Props>(function NodeActions$({
           />
         </Tooltip>
       ) : null}
-      <MenuSurfaceAnchor>
-        <Menu open={isMenuOpen} onClose={() => setMenuOpen(false)}>
-          <MenuItem onClick={removeNode}>
-            <Icon icon="delete" /> Remove
+      <SimpleMenu
+        handle={<IconButton icon="more_vert" />}
+        onOpen={() => setMenuOpen(true)}
+        onClose={() => setMenuOpen(false)}
+      >
```

Comment:
no hoistToBody for this one?

Reply:
Because the NodeActions sometimes dissapears on hover, it flashes in and out of the top of the viewer, so it's best that this does not get hoisted. I'll add a comment